client/packages/common/src/intl/locales/en/common.json

```diff
@@ -1399,6 +1400,7 @@
   "label.unacknowledged": "Unacknowledged",
   "label.unit": "Unit",
   "label.unit-name": "Unit name",
+  "label.unit-of-packs": "Unit of packs (e.g. kg, L)",
```

Comment:
Had some early feedback on this, We think probably just translating this a `Units` is better, and ideally should populate from the item units.
We think the concept is to allow the purchasing of say `Bottles` from the suppier but you manage stock in `ml` in mSupply